label_studio/utils/schema/default_config.json

```diff
@@ -17,7 +17,7 @@
 
     "!ml_backend": {
         "url": "http://localhost:9090",
-        "model_name": "my_super_model"
+        "name": "my_super_model"
```

Comment:
Hi, @guneemwelloeux ! Thank's for pointing to this. I believe that this parameter is unused due to the key `"!ml_backend"` (with exclamation). It was preliminary just for demo purpose, and could be deleted How do you get this error in runtime?